Wikipedia/Code/AppDelegate.m

```diff
@@ -115,6 +116,17 @@ - (void)resumeAppIfNecessary {
     }
 }
 
+- (BOOL)shouldShowLastReadArticleOnResume {
+    NSDate *resignActiveDate = [[NSUserDefaults wmf] wmf_appResignActiveDate];
+    if (!resignActiveDate) {
+        return NO;
+    }
+    NSDate *becomeActiveDate = [[NSUserDefaults wmf] wmf_appBecomeActiveDate];
+    NSDate *cutoffDate = [[NSCalendar wmf_utcGregorianCalendar] nextDateAfterDate:resignActiveDate matchingHour:5 minute:0 second:0 options:NSCalendarMatchStrictly];
```

Comment:
At some point we may want to round up the various constant values throughout the app (like this `hour` setting here) and consolidate them into a `constants` plist or some other structure... maybe a nested Swift enum?